src/cmds/openapi/index.ts

```diff
@@ -86,6 +92,10 @@ export default class OpenAPICommand extends Command {
       );
     }
 
+    if (create && id) {
+      Command.warn("We'll be using the `--create` option , so the `--id` parameter will be ignored.");
```

Comment:
Wonder if in this case we should have an "are you sure you want to do this" prompt or maybe them manually supplying the create arg is enough consent.

Reply:
yeah I went back and forth on this and figured that the `--id` parameter causes some confusion (see https://github.com/readmeio/rdme/issues/568) whereas the word `create` should hopefully be intuitive enough. If we get negative feedback then we can revisit.